src/config.js

```diff
@@ -38,7 +38,9 @@ const getIdleTimeout = (input) => {
 };
 
 const herokuConfig = {
-  topics: process.env.TOPICS ? process.env.TOPICS.split(',').map((string) => string.trim()) : [],
+  prefix: process.env.KAFKA_PREFIX,
+  topics: process.env.TOPICS ? process.env.TOPICS.split(',').map((string) =>
```

Comment:
need to make sure you don't end up with a topic which is just the prefix, i.e. if string is empty or null